apps/src/code-studio/components/progress/TeacherUnitOverview.tsx

```diff
@@ -239,6 +238,13 @@ const initializeRedux = (
     dispatch(setStudentDefaultsSummaryView(false));
```

Comment:
Can you tell me a bit about this one?  Why, if `unitData.student_detail_progress_view` is `true` do we `setStudentDefaultsSummaryView(false)`

Reply:
That is a very good question lol. I don't know. I copied the logic from https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/studio/pages/scripts/show.js#L82

Unfortunately I think we need to keep the backwards logic to make it compatible with the original. But could you add a comment explaining why we kept it backwards.